pkg/daemon/discover/discover.go

```diff
@@ -51,18 +51,32 @@ var (
 	cmName          string
 	cm              *v1.ConfigMap
 	udevEventPeriod = time.Duration(5) * time.Second
+	useCVInventory  bool
 )
 
-func Run(context *clusterd.Context, probeInterval time.Duration) error {
+type CephVolumeInventory struct {
+	Path      string `json:"path"`
+	Available bool   `json:"available"`
+	SysAPI    struct {
```

Comment:
So the question is: Is Explicitly listing all field in `c-v inventory` ok? @jan--f thoughts? 

We will need the full `c-v inventory`, because we already have use cases:

1. Having `ceph orchestrator device ls [-f json]` being just a aggregation of c-v inventories for all hosts completely removes the orchestrator's inventory data structure. And I'm a big fan of removing redundant or duplicated data structures. This is strongly related to  https://github.com/ceph/ceph/pull/30662#issuecomment-537714329
2. The conversion from Drive Groups to OSDs is important for @bk201 's dashboard work. 
https://github.com/ceph/ceph/pull/28896 will be moved to python-ceph-common and contains a function similar to
    ```python
    def create_osds(drive_group_spec, inventory) -> DriveSelection
    ```
    where `create_osds` uses arbitrary data from inventory.

Reply:
Not sure I understand, this only lists what rook cares about iiuc.

Reply:
full c-v inventory included